packages/collections/specs/unit/base/Sequence.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { Sequence } from '../../../..';
+import { Sequence } from '../../../src/base/Sequence';
```

Comment:
In test files please import entities from package's `index.ts`. So that we can be sure that entity accessible from outside and works correctly.

Reply:
Got it. Updated all the specs :ok_hand:
Please take a look